src/elements/pages/dev/game/game-banner.ts

```diff
@@ -15,7 +16,7 @@ export default class DevGameBanner extends LitElement {
 	renderGameIcon(game: cloud.GameFull): TemplateResult {
 		return html`
 			<lazy-img
-				class="mx-auto w-28 h-28"
+				class="mx-auto w-28 h-28 z-10"
```

Comment:
That could interfere with other elements on the page, what was the intent of introducing such high z-index?

Reply:
Z-index was meant to help make sure that the game's icon and name were not being affected by the opacity of the custom image the user uploads. Basically, I have it make the custom banner slightly transparent with a gray overlay so that there is better contrast between the background and text. I don't see this being a huge issue, as the only thing that it might interfere with would be the nav-bar (which has an even higher z-index).